### ISSUE TITLE
"WebPort80Protocol parameter has been deprecated" fails deploy

### ISSUE THREAD

**Issue by contra** [convox/rack#1008]
Running `convox deploy` after updating the rack fails with:

```
Promoting RGQJQCMAZWS...
ERROR: WebPort80Protocol parameter has been deprecated. Please set the convox.port.80.protocol label instead
```

I've never explicitly heard of or passed in any "WebPort80Protocol" parameter, so this error seems like a bug in the latest rack update.

Adding a `convox.port.80.protocol` label to docker-compose does not fix the issue. I guess the only workaround is to revert the rack version.

```
> convox -v
client: 20160805175609
server: 20160805135351 (console.convox.com)
> convox rack
Name     redacted-staging
Status   running
Version  20160805135351
Region   us-east-1
Count    5
Type     t2.small
```

( Paying customer, if that matters for support reasons )

**Comment by contra** [comment]
Caused by https://github.com/convox/rack/pull/983

**Comment by ddollar** [comment]
Fixed in #1004
